omero/scientists/index.html

```diff
@@ -21,42 +21,54 @@ <h3 class="header-red text-center">OMERO is designed with scientists in mind</h3
                 
         <div class="row small-up-12 medium-up-3">        
             <div class="column">
+                <a href="{{ site.baseurl }}/omero/import/">
                 <i class="icon-fa-red fa fa-upload fa-2x"></i>
+                </a>
                 <div class="omero-key-feature-container">
                     <h4>Import Data</h4>
                     <p>Upload your data via the desktop app or command line, use our custom dropbox tool, or have your facility manager import it for you</p>
                 </div>
             </div>
             <div class="column">
+                <a href="{{ site.baseurl }}/omero/organize/">
                 <i class="icon-fa-red fa fa-sitemap fa-2x"></i>
+                </a>
                 <div class="omero-key-feature-container">
                     <h4>Organize</h4>
                     <p>View and organize your data from anywhere you have internet access</p>
                 </div>
             </div>
             <div class="column">
+                <a href="{{ site.baseurl }}/omero/view/">
                 <i class="icon-fa-red fa fa-eye fa-2x"></i>
+                </a>
                 <div class="omero-key-feature-container">
                     <h4>View</h4>
                     <p>Work with your images using the OMERO.insight desktop app, with OMERO.web in your browser, or from 3rd party software</p>
                 </div>
             </div>
             <div class="column">
+                <a href="{{ site.baseurl }}/omero/analyze/">
                 <i class="icon-fa-red fa fa-line-chart fa-2x"></i>
+                </a>
                 <div class="omero-key-feature-container">
                     <h4>Analyze</h4>
                     <p>Use ImageJ, MATLAB, R, scripts and other tools to run analysis on data in OMERO and save results on the server</p>
                 </div>
             </div>
             <div class="column">
+                <a href="{{ site.baseurl }}/omero/organize/">
```

Comment:
Should this go out to http://help.openmicroscopy.org/sharing-data.html?

Reply:
I'd thought it'd be confusing if we had one going to help when all the others go to feature sections within the website and then link out to help or the docs from there